src/test-utilities/legacy.tsx

```diff
@@ -151,15 +151,13 @@ export function polarisContextReactWrapper<P, S>(
     }
 
     return (
-      <React.StrictMode>
```

Comment:
Why did you remove Strictmode, wasn't this the reason for this PR?

Reply:
Enzyme still uses it 😢 Although `@shopify/react-testing` and storybook don't 😄